cmd/robot/build.go

```diff
@@ -166,19 +166,30 @@ func (v *buildUploadVerb) process(ctx context.Context, id string) error {
 	return nil
 }
 
-func zipFile(zip *zip.Writer, zipVirtualPath string, filePath file.Path) error {
+func zipFile(zipWriter *zip.Writer, zipVirtualPath string, filePath file.Path) error {
```

Comment:
For better or for worse, [go prefers short, often single letter variable names](https://talks.golang.org/2014/names.slide#7). `w` would be completely acceptable for `zipWriter` here. Most of the code in this PR uses names that are far more verbose than we'd normally have, but I'm not going to ask you to change it. Just a FYI for future reference.